pkg/providers/aws/credentials.go

```diff
@@ -98,9 +99,9 @@ var (
 				"rds:RemoveTagsFromResource",
 				"rds:ApplyPendingMaintenanceAction",
 				//"sts:GetCallerIdentity",
-				"iam:CreateServiceLinkedRole",
-				"cloudwatch:ListMetrics",
-				"cloudwatch:GetMetricData",
+				//"iam:CreateServiceLinkedRole", // Only needed for CloudWatch alarms (not used)
+				//"cloudwatch:ListMetrics",	// Only needed for metric discovery (not used)
```

Comment:
I would be inclined to leave ListMetrics in as it is referenced in the interface

Reply:
something up with metrics when deployed on rhoam

<img width="1900" height="261" alt="image" src="https://github.com/user-attachments/assets/5c67f2ff-ddeb-49be-bdc6-57e0c4cefc4c" />